pkg/kubectl/cmd/create.go

```diff
@@ -53,12 +54,13 @@ func NewCmdCreate(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	options := &CreateOptions{}
 
 	cmd := &cobra.Command{
-		Use:     "create -f FILENAME",
+		Use:     "create [ -f FILENAME | -F FILES-FROM ]",
```

Comment:
We need to reserve that for --follow, probably (which in certain contexts is much more common)